Src/ILGPU/Runtime/Cuda/CudaIpcMemoryBuffer.cs

```diff
@@ -0,0 +1,173 @@
+﻿// ---------------------------------------------------------------------------------------
+//                                        ILGPU
+//                        Copyright (c) 2017-2023 ILGPU Project
+//                                    www.ilgpu.net
+//
+// File: CudaIpcMemoryBuffer.cs
+//
+// This file is part of ILGPU and is distributed under the University of Illinois Open
+// Source License. See LICENSE.txt for details.
+// ---------------------------------------------------------------------------------------
+
+using ILGPU.Resources;
+using System;
+using static ILGPU.Runtime.Cuda.CudaAPI;
+
+namespace ILGPU.Runtime.Cuda
+{
+    /// <summary>
+    /// Represents an unmanaged Cuda buffer.
```

Comment:
ISSUE: Please tidy up the comments in this file. e.g. "This represents a Cuda IPC buffer"

Reply:
I made some changes, how does it look now? I left comments of code identically to `CudaMemoryBuffer` alone, as it was the original template for the class.